modules/BOSH.Sysprep/BOSH.Sysprep.psm1

```diff
@@ -401,7 +401,7 @@ function Invoke-Sysprep()
 
   $OsVersion = Get-OSVersion
 
-  # WARN WARN: this should be removed when Microsoft fixes this bug
+  # WARN WARN: this Should -Be removed when Microsoft fixes this bug
```

Comment:
This is funny but maybe we shouldn't leave it in

Reply:
Fixed